src/formatter.ts

```diff
@@ -19,6 +19,16 @@ import { isNote, isStaveNote } from './typeguard';
 import { defined, log, midLine, RuntimeError } from './util';
 import { Voice } from './voice';
 
+// Default softmax.  Can be set on measure level with FormatterOptions
+// or globally by changing the default
+export let softmaxDefault = 100;
+export const setSoftmaxDefault = (value: number) => {
+  softmaxDefault = value;
+};
+export const getSoftmaxDefault = () => {
+  return softmaxDefault;
+};
+
```

Comment:
I think that those two function should be members of `Flow`.

Reply:
I get a lint error if softmax and accessor functions are not in the same file.  I can move both to flow, or possibly comment away the lint error.

Reply:
My proposal is to do it like `setMusicFont` / `getMusicFont`
```
setSoftmaxDefault (value: number) {
  Tables.SOFTMAX_DEFAULT = value;
};
getSoftmaxDefault() {
  return Tables.SOFTMAX_DEFAULT;
};
```
in `Flow` or set/getSoftmaxFactor if we agree in the discussion above.

Reply:
@rvilarl  see above, I tried this but it introduced a circular dependency when 'formatter' is dependent on 'flow'.  I couldn't think of an easy way around this.

Reply:
@AaronDavidNewman did you see the PR that I made to your repository?